community/index.md

```diff
@@ -101,6 +100,7 @@ Other Scala-related Discord servers include:
 
 * **[Scalameta](https://discord.gg/RFpSVth)**: Scalameta-based tooling: Metals, Scalameta, Scalafix, Scalafmt, and Mdoc
 * **[Typelevel](https://discord.gg/XF3CXcMzqD)**: about the Typelevel ecosystem for pure-functional programming in Scala
+* **[ScalaItaly](https://discord.gg/8wadTgcZVt)**: the Italian Scala community Discord server
```

Comment:
I'm not sure the position of the link should change (you can mention that it is Discord, like the `scala_ru` link mentions it is Telegram)

Reply:
Agree

Reply:
Okay, I will, move it there, thanks

Reply:
I fixed it with the last commit. (It's worth squashing IMHO, I had to fix a typo)